File: Python/Medium/إزالة الرموز من النصوص.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def removeSpecialCharacters(strParam: str) -> str:
    blacklist = "!\"#$%&'()*+,./:;<=>?@[\]^`{|}~"
    result = [x for x in strParam if x not in blacklist]
    return "".join(result)

File: challenges/JavaScript/Medium/إزالة الرموز من النصوص.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function removeSpecialCharacters(strParam) {
	let str = "";
	for (let i = 0; i < strParam.length; i++) {
		if (strParam[i] == "-" || strParam[i] == "_" || strParam[i] == " " || (strParam[i] >= "a" && strParam[i] <= "z") || (strParam[i] >= "A" && strParam[i] <= "Z")) {
			str += strParam[i];
		}
	}
	return str;
}
